src/Network/HTTPClient/Capability/ICanSendHttpRequests.php

```diff
@@ -50,72 +51,75 @@ public function fetch(string $url, int $timeout = 0, string $accept_content = ''
 
 	/**
 	 * Sends a HTTP request to a given url
 	 *
-	 * @param string $method A HTTP request
-	 * @param string $url    Url to send to
-	 * @param array  $opts   (optional parameters) associative array with:
-	 *                       	 'body' => (mixed) setting the body for sending data
-	 *                           'accept_content' => (string array) supply Accept: header with 'accept_content' as the value
-	 *                           'timeout' => int Timeout in seconds, default system config value or 60 seconds
-	 *                           'cookiejar' => path to cookie jar file
-	 *                           'header' => header array
-	 *                           'content_length' => int maximum File content length
-	 *                           'auth' => array authentication settings
+	 * @param string $method         A HTTP request
+	 * @param string $url            Url to send to
+	 * @param string $accept_content supply Accept: header with 'accept_content' as the value
+	 * @param array  $opts           (optional parameters) associative array with:
+	 *                       	      'body' => (mixed) setting the body for sending data
+	 *                                'accept_content' => (string array) supply Accept: header with 'accept_content' as the value (overrides default parameter)
+	 *                                'timeout' => int Timeout in seconds, default system config value or 60 seconds
+	 *                                'cookiejar' => path to cookie jar file
+	 *                                'header' => header array
+	 *                                'content_length' => int maximum File content length
+	 *                                'auth' => array authentication settings
 	 *
 	 * @return ICanHandleHttpResponses
 	 */
-	public function request(string $method, string $url, array $opts = []): ICanHandleHttpResponses;
+	public function request(string $method, string $url, string $accept_content = HttpClientAccept::DEFAULT, array $opts = []): ICanHandleHttpResponses;
 
 	/**
 	 * Send POST request to an URL
 	 *
-	 * @param string $url     URL to post
-	 * @param mixed  $params  array of POST variables
-	 * @param array  $headers HTTP headers
-	 * @param int    $timeout The timeout in seconds, default system config value or 60 seconds
+	 * @param string $url            URL to post
+	 * @param mixed  $params         array of POST variables
+	 * @param string $accept_content supply Accept: header with 'accept_content' as the value
+	 * @param array  $headers        HTTP headers
+	 * @param int    $timeout        The timeout in seconds, default system config value or 60 seconds
 	 *
 	 * @return ICanHandleHttpResponses The content
 	 */
-	public function post(string $url, $params, array $headers = [], int $timeout = 0): ICanHandleHttpResponses;
+	public function post(string $url, $params, string $accept_content = HttpClientAccept::DEFAULT, array $headers = [], int $timeout = 0): ICanHandleHttpResponses;
```

Comment:
I feel like the accept_content parameter should be put last because we rarely need to set it for these data submission requests.

I’m on mobile so I can’t easily check, but was there any post() call without the default Accept value?

Reply:
`post` doesn't have `accept` headers, I think.

Reply:
at least I found this discussion about it https://stackoverflow.com/a/56152516

Reply:
You might be "allowed" to use that header during `post`. But I'm convinced that most systems will format their answer according to the transmitted `content-type` of the request. So in my opinion we should only add these parameter to `get`. And when we really have to use the `accept` header for some special case, then we can use it via the header array.